src/MainPage.jsx

```diff
@@ -0,0 +1,13 @@
+import { useNavigate } from "react-router-dom";
+
+const MainPage = () => {
+    const navigate = useNavigate();
+    return (
+        <div className='mainPageWrapper'>
+            <h1>To-Do List App</h1>
+            <button onClick={() => navigate('/todos')}>Let's start!</button>
```

Comment:
One consideration - instead of powering the page change with the `onClick`, wrap the `button` in a `Link` component like on your 404 page. This helps with accessibility (denoting this is a link to another page rather than doing some action).